docs/paper/paper.md

```diff
@@ -0,0 +1,132 @@
+The analysis also attaches its results, such as fit results and figures, to the data container.
+\autoref{fig:framework} summarizes this framework.
+Importantly, this framework can interface with services to store, load, and share data.
+The library of experiments includes common quantum computing experiments such as Randomized 
+Benchmarking [@Magesan2011], Quantum state and process tomography [@Banaszek1999], Quantum Volume [@Cross2019], 
+and gate error amplifying calibration sequences [@Tornow2022].
+
+`Qiskit Experiments` is based on Qiskit [@Qiskit], a general purpose Python library for programming 
+quantum computers and simulators.
+It uses many other open source packages.
+These include Numpy [@Harris2020] for fast numerical computing, lmfit [@Newville2014] to fit complex 
+models to data, CVXPY [@Diamond2016] for convex optimization, Matplotlib [@Hunter2007] for plotting, 
+and uncertainties [@Lebigot2016] to provide measurements with a mean and a standard deviation.
+
+
+# Statement of need
+
+Quantum computing processes information following the laws of quantum mechanics.
+Quantum computers, like classical computers, must be programmed to perform quantum computations. 
+A quantum computer consists of qubits which store information in 
+quantum states, along with additional hardware elements, such as resonators to couple, control, 
+and readout the qubits. 
+The different elements in the quantum hardware have properties that must be characterized to 
+calibrate the quantum gates that process the information. 
+Furthermore, the quality of these gates has to be benchmarked to measure the overall performance 
+of the quantum computer. 
+This characterization and calibration requires an extensive set of experiments and analysis routines.
+
+Quantum development software packages such as Qiskit, ReCirq [@QuantumAI], tKet [@Sivarajah2021], and
+Forest [@Smith2016] are part of the quantum stack to execute quantum circuits on hardware.
+They also enable high-level applications that abstract away the quantum hardware. 
+Forest-benchmarking [@Forest] and pyGSTi [@pyGSTi] are tailored towards benchmarking of quantum hardware.
+However, there is still a need for open-source software that enables researchers and hardware 
+maintainers to easily execute characterization and calibration experiments.
+`Qiskit Experiments` is unique in this perspective as it provides low-level characterization 
+experiments that integrate with pulse-level control [@Alexander2020].
+In addition, `Qiskit Experiments` provides a calibration framework to manage device calibration.
+This framework is usable with any hardware exposed as a Qiskit backend.
+For example, the `Qiskit Experiments` framework has been used to explore measurements without qubit 
+reset [@Tornow2022], characterize positive operator value measures [@Fischer2022], quantum 
+states [@Hamilton2022], and time-evolutions [@Greenaway2022], as well as calibrate gates [@Vazquez2022].
```

Comment:
You can also reference our paper that just came out, https://arxiv.org/abs/2303.02108, which used Qiskit Experiments to run mirror RB and QV circuits.

Reply:
Done here: [7cc3d20](https://github.com/Qiskit/qiskit-experiments/pull/1055/commits/7cc3d20c85476b161be3864f7a015d73934c569a)